packages/atomic/src/components/atomic-load-collection/atomic-load-collection.tsx

```diff
@@ -0,0 +1,67 @@
+import {Result, FoldedResultListState} from '@coveo/headless';
+import {Component, Element, h, State} from '@stencil/core';
+import {buildCustomEvent} from '../../utils/event-utils';
+import {Bindings, InitializeBindings} from '../../utils/initialization-utils';
+import {Button} from '../common/button';
+import {FoldedResultListStateContext} from '../result-lists/result-list-decorators';
+import {ResultContext} from '../result-template-components/result-template-decorators';
+
+/**
+ * The `atomic-load-collection` component allows to load the full collection for a folded result.
+ */
+@Component({
+  tag: 'atomic-load-collection',
+})
+export class AtomicLoadMoreResults {
```

Comment:
```suggestion
export class AtomicLoadCollection {
```

I would propose `atomic-load-folded-results` or `atomic-load-more-folded-results` or use `children` instead of `folded` to keep the terms similar to other components for that feature

Reply:
I must have forgotten to change it, since the name of the file and the class are different - I try to keep them the same. Will follow your suggestion!